extensions/guacamole-auth-vault/modules/guacamole-auth-vault-base/src/main/resources/translations/en.json

```diff
@@ -0,0 +1,7 @@
+{
+
+    "DATA_SOURCE_AZURE_KEYVAULT" : {
```

Comment:
Maybe copypasta?  This is the translation file from the -base module, but is referencing the Azure keyvault.  This should be in the azure-keyvault module, no?

Reply:
In this case, no. I agree it feels odd when there is only a single implementation, but this approach is what allows the translations of multiple similar extensions to not be duplicated across the codebase. For example, here's the JDBC version of this:

https://github.com/apache/guacamole-client/blob/da495eb7c3a0d0a9bba201089d1df79869d3eedc/extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-base/src/main/resources/translations/en.json#L50-L72

And here's SSO:

https://github.com/apache/guacamole-client/blob/da495eb7c3a0d0a9bba201089d1df79869d3eedc/extensions/guacamole-auth-sso/modules/guacamole-auth-sso-base/src/main/resources/translations/en.json#L3-L13